docs/style.css

```diff
@@ -1,6 +1,20 @@
 @import url("observablehq:default.css");
 @import url("observablehq:theme-air.css") (prefers-color-scheme: light);
 @import url("observablehq:theme-near-midnight.css") (prefers-color-scheme: dark);
+@import url("https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Spline+Sans+Mono:ital,wght@0,300..700;1,300..700&display=swap");
+
+:root {
+  --serif: Inter, sans-serif;
```

Comment:
It seems strange to have `--serif` point to a sans-serif font. The only place Framework uses `--serif` internally is for the body font, so we should be able to override that without changing the definition of `--serif`. (And this only affects our own documentation site, so we don’t have to worry about this as a public API change.)

Also, I’m not sure if this matters, but typically we use a much longer set of system fonts for fallback rather than just `sans-serif` (or `serif`):

```css
  --serif: "Source Serif Pro", "Iowan Old Style", "Apple Garamond", "Palatino Linotype", "Times New Roman",
    "Droid Serif", Times, serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```